ccd-definition/CaseField/CaseField.json

```diff
@@ -906,7 +906,7 @@
   {
     "CaseTypeID": "CIVIL",
     "ID": "respondent2OrganisationPolicy",
-    "Label": "Respondent 2 organisation policy",
+    "Label": "Defendant 2's legal representative",
```

Comment:
This might be a type already used by Damages. So, if we need change in label text, it is better to create a LRspec variant of this.